pkg/sql/schemachanger/scplan/internal/opgen/opgen_sequence.go

```diff
@@ -21,12 +21,27 @@ func init() {
 	opRegistry.register((*scpb.Sequence)(nil),
 		toPublic(
 			scpb.Status_ABSENT,
-			to(scpb.Status_DROPPED,
-				emit(func(this *scpb.Sequence) *scop.NotImplemented {
-					return notImplemented(this)
+			equiv(scpb.Status_DROPPED),
+			to(scpb.Status_DESCRIPTOR_ADDED,
+				emit(func(this *scpb.Sequence) *scop.CreateSequenceDescriptor {
+					return &scop.CreateSequenceDescriptor{
+						SequenceID: this.SequenceID,
+					}
 				}),
 			),
 			to(scpb.Status_PUBLIC,
+				revertible(false),
+				emit(func(this *scpb.Sequence) *scop.InitSequence {
+					var restartWtih *int64
+					if this.GetOptionalRestartWith() != nil {
+						val := this.GetRestartWith()
+						restartWtih = &val
+					}
+					return &scop.InitSequence{
+						SequenceID:  this.SequenceID,
+						RestartWith: restartWtih,
```

Comment:
Instead of having a pointer to an int, can we have an extra `UseRestartWith` boolean field instead? I'm not sure how well an *int64 serializes in YAML (which is how we render EXPLAIN diagrams) and this isn't tested. For consistency, you might want to do the same in the element model.